.changeset/sixty-parents-explode.md

```diff
@@ -0,0 +1,5 @@
+---
+"vilay": minor
```

Comment:
Since we're currently in `0.0.x` phase, we're able to make breaking changes in `patch` releases (at least that's what I understand). I'm not sure if Changesets automatically handles that, but if not, it'd be better to have it as a `patch` change.

Reply:
ok! that's fine, whatever you prefer